config/deploy.rb

```diff
@@ -29,12 +22,26 @@
 # Default value for :pty is false
 # set :pty, true
 
+# https://github.com/capistrano/bundler#usage
+# Options, with defaults:
+#set :bundle_roles, :all                                         # this is default
+#set :bundle_servers, -> { release_roles(fetch(:bundle_roles)) } # this is default
+#set :bundle_binstubs, -> { shared_path.join('bin') }            # this is default
+#set :bundle_gemfile, -> { release_path.join('MyGemfile') }      # default: nil
+#set :bundle_path, -> { shared_path.join('bundle') }             # this is default
+#set :bundle_without, %w{development test}.join(' ')             # this is default
+#set :bundle_flags, '--deployment --quiet'                       # this is default
+#set :bundle_env_variables, {}                                   # this is default
+set :bundle_binstubs, -> { shared_path.join('.binstubs') }
+set :bundle_without, 'development test'
+set :bundle_flags, '--deployment'
+
 # Default value for :linked_files is []
 # set :linked_files, %w{config/database.yml}
 
 # Default value for linked_dirs is []
 # set :linked_dirs, %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
-set :linked_dirs, %w(log run config/environments config/certs)
+set :linked_dirs, %w(.binstubs log run config/environments config/certs)
```

Comment:
OK, I'll try to separate this out (and better check there are no cron or other things that depend on having access to this bin path).